src/views/HomeView.tsx

```diff
@@ -35,10 +68,24 @@ const HomeView: React.FC<RouteComponentProps> = () => {
         ) : (
           <ErrorFallback error={newestVideosError} resetError={() => refetchNewestVideos()} />
         )}
-
-        <ErrorBoundary fallback={ErrorFallback}>
-          <StyledInfiniteVideoGrid title="More videos" skipCount={NEWEST_VIDEOS_COUNT} />
-        </ErrorBoundary>
+        {watchedVideosWithoutFollowed && watchedVideosWithoutFollowed.length >= MIN_FOLLOWED_CHANNELS_VIDEOS ? (
+          <ErrorBoundary fallback={ErrorFallback}>
+            <InfiniteFollowedChannelsVideoGrid
+              videosToExclude={watchedVideosIds}
+              title="Followed Channels"
+              createdAtGte={MIN_DATE_FOLLOWED_CHANNELS_VIDEOS}
+              channelIdIn={channelIdIn}
+            />
+          </ErrorBoundary>
+        ) : (
+          <ErrorBoundary fallback={ErrorFallback}>
+            <StyledInfiniteVideoGrid
+              title="More videos"
+              skipCount={NEWEST_VIDEOS_COUNT}
```

Comment:
As part of this PR we should also remove the `Newest videos` carousel. So we can get rid of `skipCount` then as well